src/generator.js

```diff
@@ -654,6 +654,7 @@ class Generator {
 
     let docLines = [
       this.getPropNameFromFQP(model.type) !== model.type &&
+        model.type.indexOf("schema:") !== 0 &&
```

Comment:
Minor point: I think using `!model.type.startsWith("schema:")` is more readable.

Reply:
You're right - the IE6 in me always makes me use indexOf, but this is Node so I've changed this.